.gitignore

```diff
@@ -1,2 +1,5 @@
 dist
 node_modules
+.idea
+.DS_Store
+web-ext-artifacts
```

Comment:
Do we still need this, as we are no longer using web-ext?

Reply:
Right, removed it.